onnxruntime/python/onnxruntime_pybind_state_common.h

```diff
@@ -0,0 +1,44 @@
+#include "core/framework/allocator.h"
+#include "core/framework/session_options.h"
+#include "core/common/logging/logging.h"
+#include "core/common/logging/sinks/cerr_sink.h"
+
+namespace onnxruntime {
+namespace python {
+
+using namespace onnxruntime;
+using namespace onnxruntime::logging;
+
+static const SessionOptions& GetDefaultCPUSessionOptions() {
```

Comment:
>static [](start = 0, length = 6)

this is per translation unit, probably not what we want. also definitions with internal linkage (e.g., static function) aren't allowed in header files, per [https://google.github.io/styleguide/cppguide.html#Unnamed_Namespaces_and_Static_Variables](https://google.github.io/styleguide/cppguide.html#Unnamed_Namespaces_and_Static_Variables)

Reply:
discussed with @edgchen1 offline. corrected with inline functions.